src/redux/modules/auth/middleware.js

```diff
@@ -0,0 +1,22 @@
+import AuthService from '../../../utils/AuthService';
+
+import { loginSuccess } from './actions';
+
+const authMiddleware = store => next => (action) => {
+  AuthService.handleAuthentication()
+    .then((logged) => {
```

Comment:
What is `logged`? Should call this `authResult`.

Reply:
that was mistake on my part. You are right. should be labeled as `authResult`.